health/statuscheck/plugin_impl_statuscheck.go

```diff
@@ -73,6 +75,10 @@ func (p *Plugin) Init() error {
 		LastChange:   time.Now().Unix(),
 	}
 
+	// initial empty interface status
+	var statusData []*status.InterfaceStatus_Interfaces
+	p.interfaceStat = &status.InterfaceStatus{Interfaces: statusData}
```

Comment:
You can omit setting the field `Interfaces` since it will be `nil` by default anyway.